spec/abstractops.html

```diff
@@ -1606,10 +1606,13 @@ <h1>
       <dd>It converts _argument_ to an integer representing its Number value with fractional part truncated (replacing *NaN* with 0), or throws a *RangeError* when that Number value is not finite.</dd>
     </dl>
     <emu-alg>
+      1. If _argument_ is *undefined*, throw a *RangeError* exception.
```

Comment:
I think we can remove this line — ToNumber(undefined) is NaN, so this case will be caught by the check for NaN below.